backend/app/controllers/projects.js

```diff
@@ -263,9 +263,73 @@ module.exports = {
                     Project.select().where(Project.id.equals(req.params.id))
                 );
 
-                // Update firstActivated if the status was changed from 0 to 1
-                if (parseInt(updateObj.status) === 1 && _.first(project).firstActivated === null) {
-                    updateObj.firstActivated = new Date();
+                // Ensure conditions are met if project status is being set to 1 (activated)
+                if (parseInt(updateObj.status) === 1) {
+
+                    // Check that the survey is published
+                    const product = yield thunkQuery(
+                        Product.select().from(Product).where(Product.projectId.equals(req.params.id))
+                    );
+
+                    const survey = yield common.getSurveyFromSurveyService(product[0].surveyId, req.headers.authorization);
+
+                    if (survey.body.status !== 'published') {
+                        throw new HttpError(400, 'Survey is not published, project cannot be started');
+                    }
+
+                    // check that project has at least one subject
+                    const projectUOA = yield thunkQuery(
+                        ProductUOA.select().from(ProductUOA).where(ProductUOA.productId.equals(product[0].id))
+                    );
+
+                    if (!_.first(projectUOA)) {
+                        throw new HttpError(400, 'Project contains no subjects, project cannot be started');
```

Comment:
If you're happy with a 400 error code, we could listen for it and throw a toast explaining why the user's project didn't go active.